pkgs/development/python-modules/rfeed/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, lxml
+}:
+
+buildPythonPackage rec {
+  pname = "rfeed";
+  version = "1.1.1";
+
```

```suggestion
  pyproject = true;

  build-system = [ setuptools ];
```
Help to avoid legacy build.